sql/core/src/test/resources/sql-tests/inputs/subquery/in-subquery/in-group-by.sql

```diff
@@ -0,0 +1,117 @@
+ ('t3a', 6, 12, 10, 15, 20, 20.00, timestamp(date("2014-05-04")), date("2014-05-04")),
+ ('t1b', 10, 12, 219, 17, 25, 26.00, timestamp(date("2014-05-04")), date("2014-05-04")),
+ ('t1b', 10, 12, 19, 17, 25, 26.00, timestamp(date("2014-05-04")), date("2014-05-04")),
+ ('t1b', 8, 16, 319, 17, 25, 26.00, timestamp(date("2014-06-04")), date("2014-06-04")),
+ ('t1b', 8, 16, 19, 17, 25, 26.00, timestamp(date("2014-07-04")), date("2014-07-04")),
+ ('t3c', 17, 16, 519, 17, 25, 26.00, timestamp(date("2014-08-04")), date("2014-08-04")),
+ ('t3c', 17, 16, 19, 17, 25, 26.00, timestamp(date("2014-09-04")), date("2014-09-05")),
+ ('t1b', null, 16, 419, 17, 25, 26.00, timestamp(date("2014-10-04")), null),
+ ('t1b', null, 16, 19, 17, 25, 26.00, timestamp(date("2014-11-04")), null),
+ ('t3b', 8, null, 719, 17, 25, 26.00, timestamp(date("2014-05-04")), date("2014-05-04")),
+ ('t3b', 8, null, 19, 17, 25, 26.00, timestamp(date("2015-05-04")), date("2015-05-04"));
+
+-- correlated IN subquery
+-- GROUP BY in parent side
+-- TC 01.01
+select t1a, avg(t1b) from t1 where t1a in (select t2a from t2) group by t1a;
+-- TC 01.02
+select t1a, max(t1b) from t1 where t1b in (select t2b from t2 where t1a = t2a) group by t1a, t1d;
+-- TC 01.03
+select t1a, t1b from t1 where t1c in (select t2c from t2 where t1a = t2a) group by t1a, t1b;
+-- TC 01.04
+select t1a, sum(distinct(t1b)) from t1 where t1c in (select t2c from t2 where t1a = t2a) or
+t1c in (select t3c from t3 where t1a = t3a) group by t1a, t1c;
+-- TC 01.05
+select t1a, sum(distinct(t1b)) from t1 where t1c in (select t2c from t2 where t1a = t2a) and
+t1c in (select t3c from t3 where t1a = t3a) group by t1a, t1c;
+-- TC 01.06
+select t1a, count(distinct(t1b)) from t1 where t1c in (select t2c from t2 where t1a = t2a)
+group by t1a, t1c having t1a = "t1b";
+
+-- GROUP BY in subquery
+-- TC 01.07
+select * from t1 where t1b in (select max(t2b) from t2 group by t2a);
+-- TC 01.08
+select * from (select t2a, t2b from t2 where t2a in (select t1a from t1 where t1b = t2b) group by t2a, t2b) t2;
+-- TC 01.09
+select count(distinct(*)) from t1 where t1b in (select min(t2b) from t2 where t1a = t2a and t1c = t2c group by t2a);
+-- TC 01.10
+select t1a, t1b from t1 where t1c in (select max(t2c) from t2 where t1a = t2a group by t2a, t2c having t2c > 8);
+-- TC 01.11
+select t1a, t1b from t1 where t1c in (select t2c from t2 where t2a in
+(select min(t3a) from t3 where t3a = t2a group by t3b) group by t2c);
+-- TC 01.12, comment out pending SPARK-18863
+--select * from t1 where t1a in
+--(select min(t2a) from t2 where t2a = t2a and t2c >= 1 group by t2c having t2c in
+--(select t3c from t3 group by t3c, t3b having t2b > 6 and t3b > t2b ));
+-- TC 01.13, comment out pending SPARK-18863
+--select * from (select * from t2 where t2a in (select t1a from t1 where t1b = t2b)) t2 where t2a in
+--(select t2a from t2 where t2a = t2a and t2c > 1 group by t2a having t2c > 8);
```

Comment:
These two test cases will not be part of this test suite, right? So far, no negative cases are added to this test suite.